README.md

```diff
@@ -122,7 +123,16 @@ pip install apache-airflow[postgres,google]==1.10.12 \
  --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-1.10.12/constraints-3.7.txt"
 ```
 
+## Official source code
+
+Looking at the legal side of the software creation process, Apache Airflow is an Apache Sofware Foundation (ASF) project. This means that Apache Airflow project releases are source code releases and follow the [ASF Release Policy](http://www.apache.org/legal/release-policy.html). There is only one official release repository for Apache Airflow, like all ASF projects: [The Distribution Site of ASF](https://downloads.apache.org). Official releases are cryptographically signed by the release manager and officially voted on by the PMC members during the [Release Approval Process](http://www.apache.org/legal/release-policy.html#release-approval).```
+
+Following the ASF rules, the source packages released must be sufficient for a user to build and test the release provided they have access to the appropriate platform and tools. 
+
+Other ways of retrieving source code are "convenience" methods. For example, tagging in GitHub is a convenient way to mark the git project sources that were used to generate the official source packages. We also have binary "convenience" packages how Airflow is distributed: PyPI releases which are a convenient way of installing Apache Airflow using standard python tools or Docker Images published in the [Apache Airflow DockerHub](https://hub.docker.com/repository/docker/apache/airflow). Those are not official releases but they are built using officially released sources.
+
 ## Official Docker images
```

Comment:
I think we ccan also skip this whole section here. It was useful when we were working on the official images, but if we replace with the above mention, it should be enough.
```suggestion
```

Reply:
ok deleted that section